src/app/clusters/laundry-dryer-controls-server/laundry-dryer-controls-server.cpp

```diff
@@ -43,7 +43,7 @@ using namespace chip::app::Clusters::LaundryDryerControls::Attributes;
 using chip::Protocols::InteractionModel::Status;
 
 static constexpr size_t kLaundryDryerControlsDelegateTableSize =
-    EMBER_AF_LAUNDRY_DRYER_CONTROLS_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT;
+    EMBER_AF_LAUNDRY_WASHER_CONTROLS_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT;
```

Comment:
Why these changes to laundry dryer controls?  These don't see, right...

Reply:
Reverted these changes after fixing the zap file.